src/components/LandingPageBlocks/HeroBlock/animationScripts/animationScript.js

```diff
@@ -1,11 +1,161 @@
+// 		snippet: 
+// `@Controller("/taskWizard") 
+// class TaskWizardController {
+  
+//     @Get(produces = [MediaType.APPLICATION_JSON]) 
+//     fun index(): TaskWizard {
+//         return TaskWizard.default().build(); 
+//     }
+// }`,
+// 	},
+// ];
+
+// const writeComment = (txt) => {
+// 	if (titleFlag === 0) {
+// 		writeCode(codeSnippets[title_index].snippet);
+// 	}
+// 	if (titleFlag < txt.length) {
+// 		document.getElementById(
+// 			"comment_block-animation-txt"
+// 		).innerHTML += txt.charAt(titleFlag);
+// 		titleFlag++;
+// 		setTimeout(() => writeTitle(txt), titleSpeed);
+// 	}
+// };
+
+// const removeComment = (txt) => {
+// 	if (titleFlag === txt.length) {
+// 		document.getElementById(
+// 			"code_block-animation-title"
+// 		).innerHTML = txt = txt.slice(0, titleFlag - 1);
+// 		titleFlag--;
+// 		setTimeout(() => removeTitle(txt), 35);
+// 	} else {
+// 		if(title_index === codeSnippets.length-1){
+// 			title_index = 0;
+// 		} else {
+// 			title_index++
+// 		}
+// 		setTimeout(() => animationLogic(), 10);
+// 	}
+// };
+
+
+// const writeCode = (txt) => {
+// 	if (codeFlag < txt.length) {
+// 		document.getElementById("code_block-animation-txt").innerHTML += txt.charAt(
+// 			codeFlag
+// 		);
+// 		codeFlag++;
+// 		setTimeout(() => writeCode(txt), codeSpeed);
+// 	} else if (codeFlag === txt.length) {
+// 		setTimeout(() => removeCode(txt), 5000);
+// 	}
+// };
+
+// const removeCode = (txt) => {
+// 	if (codeFlag === txt.length && txt.length !== 0) {
+// 		txt = txt.replace(/</g, "&lt;");
+// 		codeFlag = txt.length;
+// 		document.getElementById(
+// 			"code_block-animation-txt"
+// 		).innerHTML = txt = txt.slice(0, codeFlag - 1);
+// 		codeFlag--;
+// 		setTimeout(() => removeCode(txt), 10);
+// 	} else {
+// 		setTimeout(() => animationLogic(), 50);
+// 	}
+// };
+
+// const writeTitle = (txt) => {
+// 	if (titleFlag === 0) {
+// 		writeCode(codeSnippets[title_index].snippet);
+// 	}
+// 	if (titleFlag < txt.length) {
+// 		document.getElementById(
+// 			"code_block-animation-title"
+// 		).innerHTML += txt.charAt(titleFlag);
+// 		titleFlag++;
+// 		setTimeout(() => writeTitle(txt), titleSpeed);
+// 	}
+// };
+
+// const removeTitle = (txt) => {
+// 	if (titleFlag === txt.length) {
+// 		document.getElementById(
+// 			"code_block-animation-title"
+// 		).innerHTML = txt = txt.slice(0, titleFlag - 1);
+// 		titleFlag--;
+// 		setTimeout(() => removeTitle(txt), 35);
+// 	} else {
+// 		if(title_index === codeSnippets.length-1){
+// 			title_index = 0;
+// 		} else {
+// 			title_index++
+// 		}
+// 		setTimeout(() => animationLogic(), 10);
+// 	}
+// };
+
+// const animationLogic = () => {
+// 	if (titleFlag < codeSnippets[title_index].title.length) {
+// 		setTimeout(() => writeTitle(codeSnippets[title_index].title), 10);
+// 	} else if (titleFlag === codeSnippets[title_index].title.length) {
+// 		setTimeout(() => removeTitle(codeSnippets[title_index].title), 50);
+// 	}
+// };
```

Comment:
It's a pretty bad practice to have commented out code in the master branch. Your peers wouldn't know what to do with it and it looks overwhelming. If the code is not needed then it should be removed. If you need it for personal purpose, then it's better to add it to your personal notes.

Reply:
ouh hell. i forgot about it lol